app/controllers/pilot_controller.rb

```diff
@@ -13,4 +13,10 @@ def registrations
         .where(location_id: current_user.team.locations.map(&:id))
         .group_by { |r| r.location.name }
   end
+
+  def download
+    registrations = Registration.where(location: current_user.team.locations)
```

Comment:
Feels like this should be a scope in the locations Pundit policy. Not urgent though, puntable.